src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Swish/SwishPaymentsResource.cs

```diff
@@ -1,27 +1,28 @@
-﻿using System;
-using System.Net.Http;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.SwishPayments
-{
-    public class SwishPaymentsResource : ResourceBase, ISwishPaymentsResource
-    {
-        public SwishPaymentsResource(HttpClient httpClient) : base(httpClient)
-        {
-        }
-
-        public Task<SwishPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
-        {
-            if (id == null)
-                throw new ArgumentNullException(nameof(id));
-
-            return SwishPayment.Get(id, this.httpClient, GetExpandQueryString(paymentExpand));
-        }
-
-        public Task<SwishPayment> Create(SwishPaymentRequest paymentRequest,
-                                                            PaymentExpand paymentExpand = PaymentExpand.None)
-        {
-            return SwishPayment.Create(paymentRequest, this.httpClient, GetExpandQueryString(paymentExpand));
-        }
-    }
-}
+﻿using SwedbankPay.Sdk.Payments.SwishPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class SwishPaymentsResource : ResourceBase, ISwishPaymentsResource
+    {
+        public SwishPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public Task<ISwishPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)